.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
Do we really need this file in the repository?

Reply:
@XVincentX This will pop up a dialog driving the user to install the `editorconfig` extension to try and keep the style clean by automatically applying fixups on save (trimming trailing whitespaces, ...). Of course, that will only work with vscode.

cf. https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions

If you don't think that adds any value, I'll drop it.

Reply:
Yes, I am aware of the feature. I have always been against IDE specific files in the repository, but I do not mind if you want to keep it here.

I would personally remove it.